.github/workflows/php-qa.yml

```diff
@@ -13,7 +13,7 @@ on:
             - '**workflows/php-qa.yml'
             - '**.php'
             - '**phpcs.xml.dist'
-            - '**psalm.xml'
+            - '**phpstan.neon.dist'
```

Comment:
There is another one in line 9.

Reply:
Searched now through whole code base and replaced all `psalm`-specific things. It seems like there were a few more. PHPStan does support `@psalm-*`-annotations, but I replaced them as well. 👍🏻 

See: https://github.com/inpsyde/modularity/pull/57/commits/25d32c260a08c511363810b40692c1970edcad79